UC_test/ramp_test.json

```diff
@@ -1 +1 @@
um and Minimum storage capacity in p.u.-hr","valid_range":{"max":null,"min":0.0}},{"name":"rating","data_type":"Float64","null_value":"0.0"},{"name":"activepower","data_type":"Float64","null_value":"0.0"},{"name":"inputactivepowerlimits","data_type":"Min_Max","null_value":"(min=0.0, max=0.0)","validation_action":"error","valid_range":{"max":null,"min":0.0}},{"name":"outputactivepowerlimits","data_type":"Min_Max","null_value":"(min=0.0, max=0.0)","validation_action":"error","valid_range":{"max":null,"min":0.0}},{"name":"efficiency","data_type":"NamedTuple{(:in, :out), Tuple{Float64, Float64}}","null_value":"(in=0.0, out=0.0)","validation_action":"warn","valid_range":{"max":1.0,"min":0.0}},{"name":"reactivepower","data_type":"Float64","null_value":"0.0","validation_action":"warn","valid_range":"reactivepowerlimits"},{"name":"reactivepowerlimits","data_type":"Union{Nothing, Min_Max}","null_value":"(min=0.0, max=0.0)"},{"name":"internal","data_type":"InfrastructureSystemsInternal"}],"struct_name":"GenericBattery","supertype":"Storage"},{"fields":[{"name":"name","data_type":"String","null_value":"init"},{"name":"contributingdevices","data_type":"Vector{Device}","null_value":"[ThermalStandard(nothing)]","comment":"devices from which the product can be procured"},{"name":"timeframe","data_type":"Float64","null_value":"0.0","validation_action":"error","comment":"the relative saturation timeframe","valid_range":{"max":null,"min":0.0}},{"name":"internal","data_type":"InfrastructureSystemsInternal"}],"docstring":"Data Structure for a proportional reserve product for system simulations.","struct_name":"ProportionalReserve","supertype":"Reserve"},{"fields":[{"name":"name","data_type":"String","null_value":"init"},{"name":"contributingdevices","data_type":"Vector{Device}","null_value":"[ThermalStandard(nothing)]","comment":"devices from which the product can be procured"},{"name":"timeframe","data_type":"Float64","null_value":"0.0","validation_action":"error","comment":"the relative saturation timeframe","valid_range":{"max":null,"min":0.0}},{"name":"requirement","data_type":"Float64","null_value":"0.0","comment":"the required quantity of the product should be scaled by a Forecast"},{"name":"internal","data_type":"InfrastructureSystemsInternal"}],"docstring":"Data Structure for the procurement products for system simulations.","struct_name":"StaticReserve","supertype":"Reserve"},{"fields":[{"name":"name","data_type":"String","null_value":"init"},{"name":"contributingdevices","data_type":"Vector{Device}","null_value":"[ThermalStandard(nothing)]"},{"name":"timeframe","data_type":"Float64","null_value":"0.0","validation_action":"error","valid_range":{"max":null,"min":0.0}},{"name":"requirement","data_type":"TimeSeries.TimeArray","null_value":"[]"},{"name":"internal","data_type":"InfrastructureSystemsInternal"}],"struct_name":"Transfer","supertype":"Service"}]},"basepower":100.0,"runchecks":true,"internal":{"uuid":{"value":"ccf147bd-3373-4c73-bb5a-87812ddb8616"}}}
```

Comment:
I don't know how these are used but they won't work any more.

Reply:
These are old and can be removed